bot/constants.py

```diff
@@ -25,6 +25,11 @@ class Channels(NamedTuple):
     devlog = int(os.getenv("CHANNEL_DEVLOG", 789431367167377448))
 
 
+class Emojis(NamedTuple):
+    cucumber = "\U0001f952"
+    invalid = "\u274c"
```

Comment:
suffix `emoji` as it is required by [Smart Config](https://github.com/python-discord/smartconfig), apparently. 
```suggestion
    cucumber_emoji = "\U0001f952"
    invalid_emoji = "\u274c"
```

Reply:
alrighty